renovate.json

```diff
@@ -0,0 +1,40 @@
+{
+    "extends": [
+        "config:base",
+        ":separateMajorReleases",
+        ":combinePatchMinorReleases",
+        ":ignoreUnstable",
+        ":prImmediately",
+        ":updateNotScheduled",
+        ":automergeDisabled",
+        ":autodetectPinVersions"
+    ],
+    "packageRules": [
+        {
+            "packagePatterns": [
+                "*"
+            ],
+            "semanticCommitType": "deps",
+            "semanticCommitScope": null
+        },
+        {
+            "packagePatterns": [
+                "^org.springframework.boot:spring-boot-starter-test"
+            ],
+            "semanticCommitType": "test",
+            "semanticCommitScope": "deps"
+        },
+        {
+            "packagePatterns": [
+                "^com.google.cloud:google-cloud-"
+            ],
+            "ignoreUnstable": false
+        }
+    ],
+    "labels": [
```

Comment:
is there a reason to choose the auto-merge bot over the built-in GH automerge?

Reply:
A couple reasons:
* I am following the other repos patterns
* I'm not sure if we are allowed to enable the native feature
* Maybe 1 or 2 less clicks if the label is automatically added vs needed to set automerge per PR